src/Form/Request/TwoFactoryAuthEnableType.php

```diff
@@ -20,7 +20,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ->add('secret', TextType::class, [
                 'property_path' => 'googleAuthenticatorSecret',
             ])
-            ->add('auth_code', TextType::class, [
+            ->add('authCode', TextType::class, [
```

Comment:
snake_case for attribute is more readable into json

Reply:
On project start was arrangement about camelCase. And all old stuff corresponds to it.

Reply:
camelCase for property on backend or class name is ok, but this is frontend attribute into form, but ok this is not critical